runtime/v2/model/src/main/java/com/walmartlabs/concord/runtime/v2/parser/TriggersGrammar.java

```diff
@@ -41,6 +41,27 @@
 
 public final class TriggersGrammar {
 
+    @SuppressWarnings("unchecked")
+    private static final Parser<Atom, Map<String, Object>> githubTriggerRepositoryInfoItem =
+            betweenTokens(JsonToken.START_OBJECT, JsonToken.END_OBJECT,
+                    with(ImmutableMap::<String, Object>builder,
+                            o -> options(
+                                    optional("repositoryId", regexpVal.map(v -> o.put("repositoryId", v))),
+                                    optional("repository", regexpVal.map(v -> o.put("repository", v))),
+                                    optional("projectId", regexpVal.map(v -> o.put("projectId", v))),
+                                    optional("branch", regexpVal.map(v -> o.put("githubHost", v))),
```

Comment:
`branch` on the left side, but `githubHost` on the right - a typo/copypasta perhaps?

Reply:
oops, copypasta.
fixed.